extensions/container/packages/extension/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "container",
+  "displayName": "Container Extension",
+  "description": "Allows to interact with container engines (docker, podman, etc.)",
+  "version": "0.0.1-next",
+  "icon": "icon.png",
+  "publisher": "kortex",
+  "license": "Apache-2.0",
+  "engines": {
+    "kortex": "^0.0.1"
+  },
+  "main": "./dist/extension.js",
+  "source": "./src/extension.ts",
+  "scripts": {
+    "build": "vite build && node ./scripts/build.js",
+    "test": "vitest run --coverage",
+    "test:watch": "vitest watch --coverage",
+    "watch": "vite build --watch"
+  },
+  "dependencies": {
```

Comment:
issue: these dependencies are not up to date:

the latest versions are: 

```json

  "dependencies": {
    "dockerode": "^4.0.9",
    "inversify": "^7.10.4",
    "stream-json": "^1.9.1",
    "zod": "^4.1.12"
  },
  "devDependencies": {
    "@kortex-app/api": "workspace:*",
    "@kortex-app/container-extension-api": "workspace:*",
    "@types/dockerode": "^3.3.45",
    "adm-zip": "^0.5.16",
    "mkdirp": "^3.0.1",
    "msw": "^2.12.0",
    "vite": "^7.2.2",
    "vitest": "^4.0.7"
  }
```

Reply:
I'll remove the dependencies because it has been one month that the original PR is opened
https://github.com/kortex-hub/kortex/pull/507
so if I do the update of the dependency each day, it's taking time for nothing